openmp/libomptarget/test/unified_shared_memory/close_enter_exit.c

```diff
@@ -4,7 +4,10 @@
 // UNSUPPORTED: clang-6, clang-7, clang-8, clang-9
 
 // Fails on amdgpu with error: GPU Memory Error
+// Fails on nvptx with error: an illegal memory access was encountered
 // XFAIL: amdgcn-amd-amdhsa
+// XFAIL: nvptx64-nvidia-cuda
```

Comment:
probably nvptx fixed the issue in some version. this can constantly pass on my local machine.